tox.ini

```diff
@@ -9,7 +9,7 @@ tox_pip_extensions_ext_pip_custom_platform = true
 tox_pip_extensions_ext_venv_update = true
 
 [testenv]
-basepython = /usr/bin/python2.7
+basepython = python3.6
```

Comment:
this also gets inherited into all other testenvs, so you can remove this from all other testenvs